lib/instrumentation/core/http-outbound.js

```diff
@@ -38,22 +37,19 @@ module.exports = function instrumentOutbound(agent, opts, makeRequest) {
     opts = copy.shallow(opts)
   }
 
+  const defaultPort = !opts.protocol || opts.protocol === 'http:' ? DEFAULT_PORT : DEFAULT_SSL_PORT
```

Comment:
Minor: 

for disambiguation... maybe rename `DEFAULT_PORT` to `DEFAULT_HTTP_PORT`.

Then we'd have `defaultPort` which could be either `DEFAULT_HTTP_PORT` or `DEFAULT_SSL_PORT` instead of having `defaultPort` which could be `DEFAULT_PORT` or `DEFAULT_SSL_PORT`.